### PR TITLE
[CSL-2135] hotfix for #2270

### PR DESCRIPTION
**Pull request by ksaric**
Original PR - https://github.com/input-output-hk/cardano-sl/pull/2270

### CODE REVIEW

File: wallet/src/Pos/Wallet/Web/Pending/Submission.hs [hunk 1]
```diff
@@ -116,16 +104,28 @@ submitAndSavePtx PtxSubmissionHandlers{..} enqueue ptx@PendingTx{..} = do
     when walletTxCreationDisabled $
         throwM $ InternalError "Transaction creation is disabled by configuration!"
 
-    ack <- submitTxRaw enqueue _ptxTxAux
-    saveTx (_ptxTxId, _ptxTxAux) `catches` handlers ack
-    addOnlyNewPendingTx ptx
-    when ack $ ptxUpdateMeta _ptxWallet _ptxTxId PtxMarkAcknowledged
+    now <- getCurrentTimestamp
+    if | PtxApplying poolInfo <- _ptxCond,
```
Comment (adinapoli-iohk) [inline wallet/src/Pos/Wallet/Web/Pending/Submission.hs]: Interestingly, this block of code from `108:116` was not part of https://github.com/input-output-hk/cardano-sl/pull/2270 -- Do you have any idea where this is coming from?

File: wallet/src/Pos/Wallet/Web/Pending/Submission.hs [hunk 2]
```diff
@@ -136,11 +136,15 @@ submitAndSavePtx PtxSubmissionHandlers{..} enqueue ptx@PendingTx{..} = do
...
-        pshOnNonReclaimable accepted e
+        pshOnNonReclaimable e
 
     reportError desc e outcome =
         logInfoS $
         sformat ("Transaction #"%build%" application failed ("%shown%" - "
                 %stext%")"%stext) _ptxTxId e desc outcome
+    reportSubmitted ack =
+        logDebug $
+        sformat ("submitAndSavePtx: transaction submitted with confirmation?: "
```
Comment (adinapoli-iohk) [inline wallet/src/Pos/Wallet/Web/Pending/Submission.hs]: This one was also added externally as it's not part of https://github.com/input-output-hk/cardano-sl/pull/2270 .

File: wallet/src/Pos/Wallet/Web/Pending/Submission.hs (hunk 1 above)
Comment (ksaric) [reply wallet/src/Pos/Wallet/Web/Pending/Submission.hs]: If I see correctly:

[CSL-2135] Cancel PTX on resubmission after 1h limit

https://github.com/input-output-hk/cardano-sl/commit/fa3d120b4121dd8b72f3fd3ec629633785da7795

File: wallet/src/Pos/Wallet/Web/Pending/Submission.hs (hunk 2 above)
Comment (ksaric) [reply wallet/src/Pos/Wallet/Web/Pending/Submission.hs]: https://github.com/input-output-hk/cardano-sl/commit/66b487addc70adf5ca61bba6abbce87df2efa2a3 but required for https://github.com/input-output-hk/cardano-sl/commit/fa3d120b4121dd8b72f3fd3ec629633785da7795

File: wallet/src/Pos/Wallet/Web/Pending/Submission.hs (hunk 1 above)
Comment (adinapoli-iohk) [reply wallet/src/Pos/Wallet/Web/Pending/Submission.hs]: Ah crap, and that was ported by you as well, but that PR is blocked by CI, effectively creating a bit of a deadlock.

**Comment by dcoutts** [comment]
This PR has now been superseded, all the changes are already included.

**Pull request closed without merging**